### ISSUE TITLE
Fix the configuring-query-thread-pool section

### ISSUE THREAD

**Issue by tombujok** [hazelcast/hazelcast-reference-manual#123]
http://docs.hazelcast.org/docs/3.6/manual/html-single/index.html#configuring-query-thread-pool

is:
```
You can change the size of the thread pool dedicated to query operations using the pool-size property. Below is an example of that declarative configuration.
```

should be:
```
You can change the size of the thread pool dedicated to query operations using the pool-size property.
Each query consumes a single thread from a Generic Operations ThreadPool on each node - let's call it the query-orchestrating thread.  That thread is blocked throughout the whole execution-span of a query on the node.

The query-orchestrating thread will use the threads from the query-thread pool in two cases:
- if you run a PagingPredicate - since each page is run as a separate task,
- if you set the QUERY_PREDICATE_PARALLEL_EVALUATION ( to true - since the predicates are evaluated in parallel.

Below is an example of that declarative configuration.
```
PagingPredicate and QUERY_PREDICATE_PARALLEL_EVALUATION should be links if possible
